Cargo.toml

```diff
@@ -54,6 +54,7 @@ tracing = "0.1.25"
 tracing-attributes = "0.1.15"
 tracing-futures = { version = "0.2.5", features = ["std", "std-future", "futures-03"]}
 uuid = { version = "0.8.2", features = ["v4"] }
+rust-crypto = "^0.2"
```

Comment:
There is also crate [md5](https://crates.io/crates/md5). Not sure what will be best.

Reply:
Considered that one I think, but it doesn't allow feeding it multiple buffers sequentially... So you would have to have the entire file in memory.

Reply:
Ook good choice then.